lib/Doctrine/DBAL/Driver/IBMDB2/DB2Statement.php

```diff
@@ -14,12 +14,6 @@
 use ReflectionObject;
 use ReflectionProperty;
 use stdClass;
-use const CASE_LOWER;
-use const DB2_BINARY;
-use const DB2_CHAR;
-use const DB2_LONG;
-use const DB2_PARAM_FILE;
-use const DB2_PARAM_IN;
```

Comment:
@carusogabriel is this reordering of `use const` and `use function` expected? Could you please point to the relevant change in the Doctrine or Slevomat coding standard? The old order at least seemed more logical (`const` and `function` blocks sorted alphabetically). I want to make sure it's not a bug.

Reply:
Please see https://github.com/doctrine/coding-standard/pull/131

Reply:
I see. The order is defined by [PSR-12](https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements). Thanks!